src/qt/locale/bitcoin_de.ts

```diff
@@ -2769,12 +2831,12 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+117"/>
         <source>Error initializing database environment %s! To recover, BACKUP THAT DIRECTORY, then remove everything from it except for wallet.dat.</source>
-        <translation type="unfinished"/>
+        <translation>Fehler beim initiallisieren der Datenbank Umgebung %s! Um wiederherzustellen BACKUP DAS VERZEICHNIS, dann entferne alles davon ausser die wallet.dat Datei. </translation>
     </message>
     <message>
         <location line="-20"/>
         <source>Set maximum size of high-priority/low-fee transactions in bytes (default: 27000)</source>
-        <translation type="unfinished"/>
+        <translation>Set maximalgrösse von hoch/tief Gebühren Transaktionen in Bytes (default:27000)</translation>
```

Comment:
better 'Setze Maximalgrösse von hoch priorisierte Transaktionen mit niedrigen Gebühren in [...]'